package.json

```diff
@@ -11,6 +11,7 @@
     "lint:fix": "eslint --ext .js,.mjs,.cjs,.jsx,.ts,.tsx,.vue src --fix",
     "prettier:check": "prettier --check .",
     "prettier:write": "prettier --write .",
+    "CC": "codeclimate analyze ./src",
```

Comment:
I would suggest to make this `lint:cc` and to include this as part of the the `lint` task script (in addition to eslint etc. etc.)

Is CodeClimate installed through dev dependencies?